examples/integrations/oci/atp-cdi/src/main/java/module-info.java

```diff
@@ -22,16 +22,19 @@
     requires java.naming;
     requires java.sql;
     requires java.ws.rs;
+
     requires jakarta.inject.api;
+    requires jakarta.enterprise.cdi.api;
```

Comment:
This is the `helidon-2.x` branch, so I think all references to `jakarta.*` must be replaced with `javax.*` instead.

Reply:
Have removed it.